### PR TITLE
status: 2023q2 FreeBSD Foundation report entry

### PR DESCRIPTION
**Pull request by Jehops**
Section authors:
- Introduction and Fundraising: Deb Goodkin <deb@freebsdfoundation.org>
- Partnerships and Research: Greg Wallace <greg@freebsdfoundation.org>
- Advocacy: Anne Dickison <anne@freebsdfoundation.org>
- OS Improvements: Joe Mingrone <jrm@freebsdfoundation.org>

Sponsored by:	The FreeBSD Foundation

### CODE REVIEW

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+link:https://www.freebsdfoundation.org/news-and-events/[Foundation News and Events] URL: link:https://www.freebsdfoundation.org/news-and-events/[]
+
+Contact: Deb Goodkin <deb@FreeBSDFoundation.org>
+
+The FreeBSD Foundation is a 501(c)(3) non-profit organization dedicated to supporting and promoting the FreeBSD Project and community worldwide.
+Donations from individuals and corporations fund and manage software development projects, conferences, and developer summits.
+We also provide travel grants to FreeBSD contributors, purchase and support hardware to improve and maintain FreeBSD infrastructure, and provide resources to improve security, quality assurance, and cluster administration efforts.
+We publish marketing material to promote, educate, and advocate for FreeBSD, facilitate collaboration between commercial vendors and FreeBSD developers, and finally, represent the FreeBSD Project in executing contracts, license agreements, and other legal arrangements that require a recognized legal entity.
+
+Last quarter we helped FreeBSD celebrate its 30th anniversary!
```
Comment (sergio-carlavilla) [inline website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:18]: Change last quarter to this quarter

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+* En-Wei Wu completed link:https://freebsdfoundation.org/project/wireless-internship/[another wireless internship] to improve and extend wtap, the net80211(4) Wi-Fi simulator.
+* Yan-Hao Wang worked on a link:https://freebsdfoundation.org/project/documentation-and-testing-internship/[documentation and testing project] to, e.g., build an online man page editor and add test cases for some userspace tools.
+* Christos Margiolis completed his link:https://freebsdfoundation.org/project/improving-the-kinst-dtrace-provider/[project to improve the kinst DTrace provider] by implementing inline function tracing and porting kinst to arm64 and riscv.
+* In preparation for FreeBSD 14.0, Muhammad Moinur (Moin) Rahman committed over 700 fixes or workarounds for ports affected by recent OpenSSL and LLVM updates.
+
+For more information about current and past Foundation-contracted work, visit the link:https://freebsdfoundation.org/our-work/projects/[Foundation Projects page].
+
+Here is a sampling of other Foundation-sponsored work completed over the last quarter:
```
Comment (sergio-carlavilla) [inline website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:132]: Change last quarter to this quarter

**Comment by sergio-carlavilla** [comment]
LGTM

**Review by lsalvadore — CHANGES_REQUESTED** [review]
A few changes and it is ready.

**Comment by Jehops** [comment]
Thanks for the review.  All the suggestions are incorporated.

**Review by lsalvadore — APPROVED** [review]
Approved, thanks.

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 3]
```diff
@@ -0,0 +1,156 @@
+ Foundation
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:1]:
```suggestion
=== FreeBSD Foundation
```

**Review by grahamperrin — CHANGES_REQUESTED** [review]
I'll take a proper look later. 

For now, I noticed that the first line is wrong. 

(Compare with <https://github.com/freebsd/freebsd-doc/blob/main/website/content/en/status/report-2023-04-2023-06/freebsd-foundation.adoc> for quarter two.)

**Comment by grahamperrin** [comment]
@Jehops better, thanks, however it should be a heading: 

`=== FreeBSD Foundation`

Superior to the three subheadings. 

Here's a PDF produced from your branch, I'll take a _quick_ look at the rest of the report: 

[789034e03a4b00e6722fb83b181d3ede9e51db90.pdf](https://github.com/freebsd/freebsd-doc/files/13121926/789034e03a4b00e6722fb83b181d3ede9e51db90.pdf)

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+The FreeBSD Foundation sponsors many conferences, events, and summits around the globe.
+These events can be BSD-related, open source, or technology events geared towards underrepresented groups.
+We support the FreeBSD-focused events to help provide a venue for sharing knowledge, working together on projects, and facilitating collaboration between developers and commercial users.
+This all helps provide a healthy ecosystem. We support the non-FreeBSD events to promote and raise awareness of FreeBSD, to increase the use of FreeBSD in different applications, and to recruit more contributors to the Project.
+We continue to add new events to our yearly roster. This July, we held a workshop and staffed a table at FOSSY, a new open source conference in Portland.
+In addition to attending and planning conferences, we are continually working on new training initiatives and updating our selection of link:https://freebsdfoundation.org/freebsd-project/resources/[how-to guides] to facilitate getting more folks to try out FreeBSD.
+
+Check out some of our advocacy work:
+
+* Held a workshop and hosted a table at link:https://sfconservancy.org/fossy/[FOSSY], July 13-16, 2023, in Portland, OR.
```
[the remark this answers is not in the document]
Comment (grahamperrin) [reply website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:75]: Global, so don't assume that OR will be immediately understandable.

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+** link:https://freebsdfoundation.org/blog/july-2023-software-development-projects-update/[July 2023 Software Development Projects Update]
+** https://freebsdfoundation.org/blog/freebsd-for-research-cheri-morello/[FreeBSD for Research: CHERI/Morello]
+** Meet the FreeBSD Google Summer of Code Students
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-soobin-rho/[Soobin Rho]
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-raghav-sharma/[Raghav Sharma]
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-sudhanshu-mohan-kashyap/[Sudhanshu Mohan Kashyap]
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-aymeric-wibo/[Aymeric Wibo]
+** link:https://freebsdfoundation.org/blog/meet-the-summer-2023-university-of-waterloo-co-op-student-naman-sood/[Meet The Summer 2023 University of Waterloo Co-Op Student: Naman Sood]
+** link:https://freebsdfoundation.org/blog/meet-freebsd-foundation-2023-summer-intern-jake-freeland/[Meet FreeBSD Foundation 2023 Summer Intern: Jake Freeland]
+* FreeBSD in the News:
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:94]:
```suggestion
* FreeBSD in the News
```

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+** link:https://freebsdfoundation.org/news-and-events/latest-news/freebsd-foundation-joins-osis-open-policy-alliance/[FreeBSD Foundation Joins OSI's Open Policy Alliance]
+** link:https://freebsdfoundation.org/news-and-events/latest-news/hackernoon-5-reasons-we-use-open-source-freebsd-as-our-enterprise-os/[Hackernoon: 5 Reasons We Use Open Source FreeBSD as Our Enterprise OS]
+** link:https://freebsdfoundation.org/news-and-events/latest-news/what-the-dev-podcast-the-evolution-of-the-freebsd-project/[What the Dev Podcast: The Evolution of the FreeBSD Project]
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:97]:
```suggestion
** link:https://freebsdfoundation.org/news-and-events/latest-news/what-the-dev-podcast-the-evolution-of-the-freebsd-project/[What the Dev Podcast: The Evolution of the FreeBSD Project].
```

A full stop at the end of the list.

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+
+We help educate the world about FreeBSD by publishing the professionally produced FreeBSD Journal.
+As we mentioned previously, the FreeBSD Journal is now a free publication.
+Find out more and access the latest issues at link:https://www.freebsdfoundation.org/journal/[].
+
+You can find out more about events we attended and upcoming events at link:https://www.FreeBSDfoundation.org/news-and-events/[].
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:103]:
```suggestion
You can find out more about events we attended, and upcoming events, at link:https://www.FreeBSDfoundation.org/news-and-events/[].
```

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+During the third quarter of 2023, 282 src, 652 ports, and 24 doc tree commits identified The FreeBSD Foundation as a sponsor.
+Some of this Foundation-sponsored work is described in separate report entries:
+
+* <<_enabling_snapshots_on_filesystems_using_journaled_soft_updates,Enabling Snapshots on Filesystems Using Journaled Soft Updates>>
+* <<_login_classes_fixes_and_improvements,Login Classes Fixes and Improvements>>
+* <<_openssl_3_in_base_-_improved,OpenSSL 3 in base - improved>>
+* <<_openstack_on_freebsd,OpenStack on FreeBSD>>
+* <<_process_visibility_security_policies,Process Visibility Security Policies>>
+* <<_simd enhancements for amd64,SIMD enhancements for amd64>>
+
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:116]: Lorenzo, I assume that these lines will be edited during collation, for links to work.

**Review by grahamperrin — CHANGES_REQUESTED** [review]

**Comment by grahamperrin** [comment]
A handful of corrections, some changes of URL to account for redirects. 

A few suggestions; take as much or little as you like.

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc (hunk 7 above)
Comment (Jehops) [reply website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc]: I don't think either of these commas is standard.  The first is before a conjunction that introduces a dependent clause.  Grammarly agrees.

Comment (grahamperrin) [reply website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc]: Commas removed, thanks. 

URL updated (one of the redirects), sorry, I might have missed this one a few hours ago.

**Comment by grahamperrin** [comment]
@Jehops thanks again. 

If you're in the desktop web view of GitHub and in the _Conversation_ tab alone, this one might have been overlooked: 

<https://github.com/freebsd/freebsd-doc/pull/285/files#r1370609053>

Pictured below, I think the word 'the' is missing: 

image

----

Maybe amongst the conversations that are hidden; more easily visible under the _Files changed_ tab.

HTH

**Comment by Jehops** [comment]
When everything looks good, please let me know.  I'll squash the commits for these corrections and add `Reviewed by:` to the commit log.

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+Contact: Deb Goodkin <deb@FreeBSDFoundation.org>
+
+The FreeBSD Foundation is a 501(c)(3) non-profit organization dedicated to supporting and promoting the FreeBSD Project and community worldwide.
+Donations from individuals and corporations fund and manage software development projects, conferences, and developer summits.
+We also provide travel grants to FreeBSD contributors, purchase and support hardware to improve and maintain FreeBSD infrastructure, and provide resources to improve security, quality assurance, and cluster administration efforts.
+We publish marketing material to promote, educate, and advocate for FreeBSD, facilitate collaboration between commercial vendors and FreeBSD developers, and finally, represent the FreeBSD Project in executing contracts, license agreements, and other legal arrangements that require a recognized legal entity.
+
+This quarter we helped FreeBSD celebrate its 30th anniversary!
+This excitement has propelled us to accelerate our efforts to move FreeBSD forward in growth and innovation, which has focused us on identifying key areas we can invest our resources.
```
[the remark this answers is not in the document]
Comment (lsalvadore) [reply website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:19]: I think you meant "to invest our efforts".

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc (hunk 8 above)
Comment (lsalvadore) [reply website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:116]: I will check them and fix them if necessary, but this is an adoc syntax that should work. See also the foundation's report from last quarter.

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc (hunk 9 above)
[the remark this answers is not in the document]
Comment (Jehops) [reply website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:19]: It's not my text, but I believe the original was intended.

File: website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+Several conferences aided these relationships, including EuroBSDCon, Open Source Summit, and All Things Open.
+We have also developed a new program to support vendor/cloud users that work with the US government.
+The program details will be announced at the FreeBSD Vendor Summit.
+
+==== Advocacy
+
+Much of our effort is dedicated to the FreeBSD Project advocacy.
+This may involve highlighting interesting FreeBSD work, producing literature and video tutorials, attending events, or giving presentations.
+The goal of the literature we produce is to teach people FreeBSD basics and help make their path to adoption or contribution easier.
+Other than attending and presenting at events, we encourage and help community members run their own FreeBSD events, give presentations, or staff FreeBSD tables.
```
[the remark this answers is not in the document]
Comment (Jehops) [reply website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc:64]: It's not my text, but the original was intended.

**Comment by grahamperrin** [comment]
@lsalvadore to re-approve? As Status Team. 

Thanks

**Comment by Jehops** [comment]
@lsalvadore approved on IRC.

**Pull request closed without merging**